docker-compose.yml

```
@@ -509,6 +509,7 @@ services:
       VIRTUAL_HOST: finder-frontend.dev.gov.uk
       PORT: 3062
     healthcheck:
+      test: "curl --silent --fail localhost:3062/healthcheck/ready || exit 1"
```

optional: consider substituting the port number
```suggestion
      test: "curl --silent --fail localhost:$PORT/healthcheck/ready || exit 1"
```

---

By adding that, the jenkins test fails.